test/runtests.jl

```diff
@@ -1,6 +1,9 @@
 using Suppressor
 using Base.Test
 
+# make sure color is off to simplify testing for output
+eval(Base, :(have_color = false))
```

Comment:
I merged this commit and fixed that issue at: https://github.com/Ismael-VC/Suppressor.jl/pull/12/files using: `use_color(enable::Bool = true) = eval(Base, :(have_color = $enable))`